src/components/shared/LessonSide/LevelSelector.tsx

```diff
@@ -1,5 +1,31 @@
-function LevelSelector(): JSX.Element {
-  return <div>Level Selector</div>;
+import '../../../styles/LessonSide.scss';
+
+interface IProps {
+  items: number;
+  current: number;
```

Comment:
I'd rename these to make it easier to know what these are. `items` sounds like it implies and array of some type of object. Instead, I'd probs call it `numLevels` or something. Similar idea with `current`